### ISSUE TITLE
Does not resolve and follow modules with webpack aliases

### ISSUE THREAD

**Issue by austincondiff** [AsaAyers/js-hyperclick#97]
For example, in my `webpack.config.js` file if I specify...

```js
resolve: {
  alias: {
    '~': path.resolve(__dirname, 'src')
  }
}
```

I'd like to be able to click into an `import` statement like this...

```js
import MyComponent from '~/components/MyComponent'
```

However currently it says that it cannot resolve `~/components/MyComponent`.

**Comment by austincondiff** [comment]
Any update on this?

**Comment by AsaAyers** [comment]
I don't have any plans to support that. https://github.com/AsaAyers/js-hyperclick/issues/58#issuecomment-283175642

**Comment by AsaAyers** [comment]
I just published v1.15 with a new feature I'm calling "Custom Resolvers". It will allow you to add a JS file to your project that can implement your aliases. [PR and documentation](https://github.com/AsaAyers/js-hyperclick/pull/105)
